data/beaconSets/btc_usd.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "BTC/USD",
+  "description": "BTC/USD",
+  "beaconSetId": "0xe9534bbc6c6084429b26b78b158f8ee65fe0a8103003066320a89ffab8156bc7",
+  "airnodeAddress": "0xF0f875aa568521FBd5246fAE7260E04d9ED6511b",
```

Comment:
A Beacon set can be composed of Beacons belonging to different Airnode addresses, so a single Airnode address doesn't make sense here

Reply:
Good catch! that was a copy+pasting issue. The actual `airnodeAddress` is used from each individual `beacon` so I'll just remove it.